ui/imports/shared/popups/addaccount/panels/DerivationPathInput.qml

```diff
@@ -107,7 +107,7 @@ Item {
         readonly property var passthroughKeys: [Qt.Key_Left, Qt.Key_Right, Qt.Key_Escape]
 
         Shortcut {
-            sequence: StandardKey.Copy
+            sequences: [StandardKey.Copy]
```

Comment:
This was yielding a warning too about the sequence being ambiguos -> the shortcut wasn't functional